packages/expo-av/android/src/main/java/expo/modules/av/AVManager.java

```diff
@@ -179,13 +184,29 @@ public void onCreate(ModuleRegistry moduleRegistry) {
       final UIManager uiManager = getUIManager();
 
       uiManager.registerLifecycleEventListener(this);
-      uiManager.runOnClientCodeQueueThread(() -> {
-        final JavaScriptContextProvider jsContextProvider = mModuleRegistry.getModule(JavaScriptContextProvider.class);
-        final long jsContextRef = jsContextProvider.getJavaScriptContextRef();
-        if (jsContextRef != 0) {
-          installJSIBindings(jsContextRef, jsContextProvider.getJSCallInvokerHolder());
-        }
-      });
+      uiManager.runOnClientCodeQueueThread(this::installBindings);
+    }
+  }
+
+  @OptIn(markerClass = FrameworkAPI.class)
+  private void installBindings() {
+    final JavaScriptContextProvider jsContextProvider = mModuleRegistry.getModule(JavaScriptContextProvider.class);
+    final long jsContextRef = jsContextProvider.getJavaScriptContextRef();
+    if (jsContextRef <= 0) {
```

Comment:
```suggestion
    if (jsContextRef != 0) {
```

nit: not sure if there's integer overflow here but maybe `!= 0` is more semantic correct